data/crac-io/crac-io-api/src/main/java/com/farao_community/farao/data/crac_io_api/CracImporters.java

```diff
@@ -29,6 +35,40 @@
     private CracImporters() {
     }
 
+    public static void cracAliasesUtil(Crac crac, Network network) {
+        // List (without duplicates) all the crac elements that need to be found in the network
+        Set<String> elementIds = new HashSet<>();
+        crac.getCnecs().forEach(cnec -> elementIds.add(cnec.getNetworkElement().getId()));
+        crac.getContingencies().forEach(contingency -> contingency.getNetworkElements().forEach(networkElement -> elementIds.add(networkElement.getId())));
```

Comment:
Don't you need to add aliases for elements of the network actions and range actions as well?